cmd/kubeapps-apis/plugins/helm/packages/v1alpha1/server.go

```diff
@@ -1027,3 +1028,30 @@ func (s *Server) fetchChartWithRegistrySecrets(ctx context.Context, chartDetails
 
 	return ch, registrySecrets, nil
 }
+
+// DeleteInstalledPackage deletes an installed package.
+func (s *Server) DeleteInstalledPackage(ctx context.Context, request *corev1.DeleteInstalledPackageRequest) (*corev1.DeleteInstalledPackageResponse, error) {
+	installedRef := request.GetInstalledPackageRef()
+	releaseName := installedRef.GetIdentifier()
+	namespace := installedRef.GetContext().GetNamespace()
+	contextMsg := fmt.Sprintf("(cluster=%q, namespace=%q)", installedRef.GetContext().GetCluster(), namespace)
+	log.Infof("+helm DeleteInstalledPackage %s", contextMsg)
+
+	// Create an action config for the installed pkg context.
+	actionConfig, err := s.actionConfigGetter(ctx, installedRef.GetContext())
+	if err != nil {
+		return nil, status.Errorf(codes.Internal, "Unable to create Helm action config: %v", err)
+	}
+
+	keepHistory := false
+	err = agent.DeleteRelease(actionConfig, releaseName, keepHistory)
```

Comment:
Yes, the UI used to allow this with a checkbox to also delete the history, because leaving the history was actually the default when you `helm uninstall`ed something, but helm then updated the default to delete the release by default. I don't remember if we switched the default or just removed the option - but the history will show.